CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ## Version 3.2.7
-* Added parameter in core section of nginx.conf via pillar
+* 3 nginx.conf parameters can be customised via pillar
```

Comment:
Leave change entries as UNRELEASED, let the person who releases the new version include them in the version number